src/Microsoft.Framework.PackageManager/Utils/Environment.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Framework.Runtime;
+
+#if DNXCORE50
+namespace Microsoft.Framework.PackageManager.Internal
+{
+    internal static class Environment
```

Comment:
Couldn't we merge this in to DnuEnvironment and just use DnuEnvironment everywhere (even to get things like Program Files which are available through System.Environment) to ensure we have one central location for these things?